app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,88 @@
+.overlay-container {
+  background: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.